apps/prairielearn/elements/pl-string-input/pl-string-input.py

```diff
@@ -244,7 +244,7 @@ def parse(element_html: str, data: pl.QuestionData) -> None:
         a_sub = "".join(a_sub.split())
 
     # Always simplify multiline characters (if they still exist)
-    a_sub = re.sub("\r*\n", "\n", a_sub)
+    a_sub = re.sub(r"\r*\n", "\n", a_sub)
```

Comment:
@eliotwrobson I don't think these should be raw strings because we do want `\` to be an escape.

Reply:
Completely right 🤦🏽 I was resolving the linter error without thinking.

Reply:
Fixed https://github.com/PrairieLearn/PrairieLearn/pull/9438/commits/c8d543acd1f2ae9ef4cf2263206fb298fbc22ddc